runners/flink/1.5/build.gradle

```diff
@@ -28,6 +28,7 @@ project.ext {
   main_resources_dirs = ["$basePath/src/main/resources"]
   test_resources_dirs = ["$basePath/src/test/resources"]
   archives_base_name = 'beam-runners-flink_2.11'
+  automatic_module_name = 'beam.runners.flink.2.11'
```

Comment:
I am not sure if this is the right module name for flink in different versions but couldn't come up with anything else. Please take extra look here. :)

Reply:
I don't know if the _2.11 path will be removed in the future I suppose that naming it with the exact version is probably better. Maybe @mxm may have a say. 

I am wondering also if the `.` in the version of the name can produce unintented consequences (package wise) in Java 11 but I don't think so.

Reply:
I think `beam.runners.flink` would suffice here. If you want to include the version, you can use the new artifact naming convention, e.g. `beam.runners.flink.1.5`

Reply:
I will keep the version as Max suggests.

Reply:
I would go without the version. No need to encode it.

Reply:
I think I misunderstood you before. 

I did some thinking - If we decide to name the modules without version `beam.runners.flink`, we will prevent users from importing 2 different versions of Flink a once*. Preventing this sounds sane now but will it always be the case?

According to my knowledge and current reasoning, the answer is "yes". Could you confirm?

*JPMS will simply refuse to start a modulepath where two modules have the same name, even if they contain different packages.

Reply:
Exactly. That's why I would prefer `beam.runners.flink`.